[Spectrum] Added SFTP section (#6318)
cloudflare/cloudflare-docs: content/spectrum/about/ftp.md

> pasv_promiscuous=YES
> ```

## SFTP

> Contrarily to FTP or FTPS, enabling Spectrum for SFTP does not require extra configuration, you can just set up a Spectrum application for SSH, select port 22 and TCP.

Reviewer: ```suggestion
Unlike FTP or FTPS, enabling Spectrum for SFTP does not require extra configuration. When setting up a Spectrum application for SSH, select port 22 and TCP.
```
